src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -27,6 +27,10 @@ def _sample_apm():
     return random.random() < getattr(settings, "SENTRY_RELAY_ENDPOINT_APM_SAMPLING", 0)
 
 
+def get_global_config():
```

Comment:
This is part of the config computation already. What do you think about moving it to the `sentry.relay` module with the rest of the project config functionality?

Reply:
i made it simply take the values from the project config computation, I thought we could avoid getting those values if global config isn't requested to save on some processing, but at least now we save on network traffic and on relay memory.  
 
whats your thoughts on that? my thinking now was that, we can do it this simple way first, and then later try to be more specific with what the `_get_cached_or_schedule` function returns later

Reply:
I'm not sure if I understand. My suggestion was to move the function into another module with the rest of the functions, not to reuse the same function itself. What do you think about separating these?